src/intl/messages.js

```diff
@@ -117,6 +117,10 @@ export const messages = {
     description: 'About application',
   },
   logOut: 'Log out',
+  noMessages: {
+    message: 'No\u00A0messages',
```

Comment:
I think here need to add some comment about \u00A0, because it is not very clear what this sign for.

Reply:
Do you think following line (122) is not enough?

Reply:
Aha, yes, sorry, I didn't saw that.